src/templates/blog-post.tsx

```diff
@@ -83,6 +85,10 @@ const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
             name: 'og:type',
             content: 'article'
           },
+          {
+            property: 'og:image',
+            content: 'ogimage'
+          },
```

Comment:
We don't need to pass `og:image` here.
cc: @RCdeWit

Reply:
Whoops, this should not have ended up here. I thought I was mucking about in a different branch. Will remove.

Reply:
Fixed! And learned a few things about rebasing along the way. 😄